### DISCUSSION TITLE
How to install MT5 on Termux Proot Distro (Debian)

### DISCUSSION THREAD

**Discussion by Ystyphn** [termux/termux-packages#18207] [Q&A] [answered]
I am trying to install MT5 on proot distro to start developing expert advisors since I don't have desktop computer. Does anyone have step by step guide on how to install MT5 on termux proot-distro?

**Comment by twaik** [comment, marked as answer]
1. This software is prebuilt and released only for x86. 
2. This software is not opensource so you can not compile aarch64 version for proot-distro.
Official site distributes install script which simply install it in Wine environment, maybe you will have luck with other tools which can emulate Win32/Win64.

**Reply by Ystyphn** [reply]
Any suggestions for Win32/Win64 emulator?

**Reply by twaik** [reply]
No. It is off-topic.
